src/components/App.js

```diff
@@ -1,29 +1,73 @@
+/* eslint-disable react-hooks/exhaustive-deps */
 import { useQuery } from '@apollo/client';
 import { GET_ALL_ITEMS } from '../api';
+import { useState, useEffect } from 'react';
+import Landing from './Landing';
+import Products from './Products';
+import About from './About';
 
 function App() {
-  const { loading, error, data } = useQuery(GET_ALL_ITEMS)
+  const [items, setItems] = useState([])
+  const [itemsForDisplay, setItemsForDisplay] = useState([])
 
-  let allItems
-
-  if (!loading) {
-    console.log('data:', data)
-    allItems = data.queryItems.map(item => {
-      return <div>
-        <h1>{item.name}</h1>
-        <p>{item.description}</p>
-        <p>{item.color}</p>
-        <p>{item.price}</p>
-      </div>
-    })
+  const getItemsForDisplay = () => {
+    let productsForDisplay = [];
+    
+    const productNames = new Set (items.map(product => product.name))
```

Comment:
never seen the new Set syntax before, is it doing something similar to spread operator?